### PR TITLE
chore: remove Intercom widget

### PR DESCRIPTION
**Pull request by bonustrack**
Removing Intercom in site messenger widget to instead redirect to our helpdesk. This is to avoid cookie storage and all tracking from Intercom in our site, this will also avoid triggering ad blockers. Unfortunately this can't be removed on Intercom side directly, so instead we showing a similar widget but with a redirection to our helpdesk.

### CODE REVIEW

File: apps/ui/src/App.vue [hunk 1]
```diff
@@ -48,4 +46,5 @@ onMounted(() => {
   <LayoutSplashScreen v-if="!whiteLabelResolved" />
   <LayoutSite v-else-if="routeName === 'site'" />
   <LayoutApp v-else />
+  <Messenger />
```
[the remark this answers is not in the document]
Comment (bonustrack) [reply apps/ui/src/App.vue:49]: Yes, its on purpose, whitelabel user may need help too

File: apps/ui/src/views/Terms.vue [hunk 2]
```diff
@@ -255,7 +255,7 @@ You agree to promptly notify the Foundation of any Claims and cooperate with the
 
 ## 15. Privacy notice and cookie declaration
```
Comment (wa0x6e) [inline apps/ui/src/views/Terms.vue:256]: Maybe also remove the cookie reference in the title, or add something in the body that says we don't track cookies

**Review by Sekhmet — COMMENTED** [review]
Intercom used to be hidden in development, can we do the same here?

```
import.meta.env.DEV
```

**Comment by bonustrack** [comment]
I think its fine to have it in local, so we have the same behavior than prod.

**Review by wa0x6e — APPROVED** [review]
tAck

**Pull request merged**